teos/watcher.py

```diff
@@ -228,6 +227,7 @@ def __init__(self, db_manager, gatekeeper, block_processor, responder, sk, max_a
         self.signing_key = sk
         self.last_known_block = db_manager.load_last_block_hash_watcher()
         self.locator_cache = LocatorCache(blocks_in_cache)
+        self.rw_lock = rwlock.RWLockWrite()
```

Comment:
Should add attribute in the class docs.
Same for `LocatorCache` (there it was already missing before this PR).